hnn_core/network.py

```diff
@@ -1127,42 +1127,76 @@ def _instantiate_drives(self, tstop, n_trials=1):
                 self.external_drives[
                     drive['name']]['events'].append(event_times)
 
-    def add_tonic_bias(self, *, cell_type=None, amplitude=None,
-                       t0=None, tstop=None):
-        """Attach parameters of tonic biasing input for a given cell type.
+    def _add_cell_type_bias(self, *, cell_type=None, amplitude,
```

Comment:
can you make this a separate function rather than have it as a method ... the more methods you add, the more complicated the `Network` class becomes and harder to refactor in the future.

Reply:
I moved it this block of code as a separate function